onboarding/mailing.py

```diff
@@ -103,3 +103,22 @@ def send_remove_acc_email(EMAIL_HOST_USER, user_email):
         html_message=html_message,
     )
 
+def answer_send_notification_email(EMAIL_HOST_USER, user_email, employee, form):
+        subject = 'Użytkownik {} wysłał formularz {}.'.format(employee, form)
+
+        html_message = render_to_string(
+            'templated_email/answer_notification.html',
+            {
+                'username': user_email,
+            }
+        )
+        plain_message = strip_tags(html_message)
+        from_email = EMAIL_HOST_USER
+        to = user_email
```

Comment:
This part is problematic for the `user.first_name` issue (https://github.com/binawork/OnlineOnboarding/pull/195/files#r595318084).
Take a look at this:
this e-mail is generated as many common ones and list of addressees has many e-mail values (`to=`),
but every recipient has another name (it is normal, isn't it :grinning: :wink: )
and there is no field called `user`. As a result we don't have and can not have a `first_name` property (besides the fact that everyone can see the addresses of others).
  
BTW: one unpleasant fact is that variable called `user_email` keeps many e-mails but it is named in singular :confused:  (it is ambiguous).